Gemfile

```diff
@@ -1,10 +1,10 @@
 source :rubygems
 
-gem "activesupport", ">= 2.3.6"
+gem "activesupport", '>= 2.3.6', '< 4.0.0'
 gem "tlsmail" if RUBY_VERSION <= '1.8.6'
 gem "mime-types", "~> 1.16"
 gem "treetop", "~> 1.4.10"
-gem "i18n", ">= 0.4.0"
+gem "i18n", '>= 0.4.0', '< 0.7.0'
```

Comment:
I understand we're limiting these since newer versions require Ruby >= 1.9.3.

However, Ruby 1.9.3+ users may use the mail lib also. Should we restrict their maximum version? It would be a very incompatible change for a stable 2.4.x release.